### PR TITLE
New Poll - Quantitative Easing

### PR DESCRIPTION
**Pull request by SebVentures**

### CODE REVIEW

File: governance/polls/Proposal - Quantitative Easing aka Manual PSM - September 14, 2020.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+This non-standard weekly Governance Poll ([FAQ](https://community-development.makerdao.com/governance/governance#is-there-more-than-one-type-of-vote)) will be active for three days beginning on Monday, September 14 at 16:00 UTC.
+
+## Review
+
+The signal request for Quantitative Easing can be found [here](https://forum.makerdao.com/t/3902) along with the results. Please review the discussion there to inform your position before voting.
+
+## Next Steps
+
+If this poll ends with a 'Yes' majority, the community will work on providing an implementation.
```
Comment (LongForWisdom) [inline governance/polls/Proposal - Quantitative Easing aka Manual PSM - September 14, 2020.md:23]: I would maybe reword this. There is no guarantee this will happen. Maybe: "If this poll ends with a 'Yes' majority, it will signal to the community that MKR Token Holders support a community implementation."

**Review by LongForWisdom — CHANGES_REQUESTED** [review]
Great job Seb, just one minor suggested revision. Thanks!

**Review by SebVentures — COMMENTED** [review]
QE - Wording on the YES case

**Review by LongForWisdom — APPROVED** [review]
Looks good. Thanks again.
